New Page on Service Terms for License Metrics

tydunne opened the pull request:
Added new page for our contractual license metrics

tydunne committed:
Create license-metrics-bundle

Does this work?

tydunne committed:
Delete content/license-metrics-bundle

deleted that

tydunne committed:
Create _index.html

I think this is creating the license-metrics-bundle folder plus the index HTML file

tydunne committed:
Update _index.html

updated index HTML file (copied from other one)

tydunne committed:
Create v1.md file

Test v1.md file copied from the other bundle

tydunne committed:
Rename v1 to v1.md

Forgot to add md to the file name

tydunne committed:
Update v1.md

Completed metrics for Messages, ODS, and CCUs with examples where relevant. Start with DataHub GIB Queried tomorrow.

tydunne committed:
Update v1.md

finalized metrics and formatting

tydunne committed:
Update _index.html

tydunne commented:
Hi @BeateRixen, looks like the bundle is not appearing as we would like: 
![image](https://github.com/user-attachments/assets/85bdf14a-6fef-4865-8a8d-8558f6dad9b5)
I couldn't quite figure out how to fix it. I edited the _index file but that didn't seem to do anything. Thanks for your help!

vaillant commented on content/service-terms/license-metrics-bundle/v1.md:
```diff
@@ -0,0 +1,110 @@
+
+## Deployment
+
+A unique instance of the Hosted Software provided by the Supplier and made accessible to the Customer. Deployments can be "Dedicated" instances or "Public Cloud" instances. 
```
Is "Dedicated" also for "on Premise"? I guess note. 
In that case, it would be nice to name these similar, either Dedicated Cloud/Public Cloud  or Dedicated/Public.

tydunne replied:
You're right, it is good to keep parity between the names. I will refer to it as "Dedicated Cloud" and "Public Cloud".

vaillant commented on content/service-terms/license-metrics-bundle/v1.md:
```diff
@@ -0,0 +1,110 @@
+## Deployment
+
+A unique instance of the Hosted Software provided by the Supplier and made accessible to the Customer. Deployments can be "Dedicated" instances or "Public Cloud" instances. 
+* Dedicated Deployments have dedicated resources provided for a single Customer. Each Dedicated Deployment includes up to 3 nodes of Cumulocity IoT Database Enterprise Edition, where each node is limited to a physical or virtual machine that has up to 128 GB of RAM allocated to such node. Cumulocity IoT Database Enterprise Edition contains third party technology and the following additional licen ...
```
IMHO this ("MongoDB License fallback") is only relevant for On Premise and not for Dedicated Cloud.

tydunne replied:
I took it directly from the existing contracts, which did include it for Dedicated Cloud. This text was given to us by the Software AG TCA team (Martin Elgert) who asked it to be put in our contracts. If we don't need it, I am fine to remove it.

vaillant commented on content/service-terms/license-metrics-bundle/v1.md:
```diff
@@ -0,0 +1,110 @@
+* Dedicated Deployments have dedicated resources provided for a single Customer. Each Dedicated Deployment includes up to 3 nodes of Cumulocity IoT Database Enterprise Edition, where each node is limited to a physical or virtual machine that has up to 128 GB of RAM allocated to such node. Cumulocity IoT Database Enterprise Edition contains third party technology and the following additional licen ...
+* Public Cloud Deployments have shared resources provided for Customers. A Public Cloud Deployment will not have data shared between Customers. 
+
+Cumulocity Dedicated Deployments can be used for Development or Testing ("Non-Production") purposes but are restricted in the scope of their usage. Dedicated Deployments may be used by the Customer solely for non-production purposes only and may not be used for the processing of live data in a production environment or any purposes other than development or testing purposes only. A Non-Production ...
```
I guess 'may not be used for the processing of live data" is simply wrong.

tydunne replied:
This text is referring to processing of data in a non-production environment, basically telling customers they cannot use it for their production data (e.g. buying a cheaper dev instance and then using it as production). The words "production environment" make this confusing, I'll change it to: "_Dedicated Deployments may be used by the Customer for non-production purposes. In this case, the Dedicated Deployment may not be used for the processing of live production data or any purposes other than development or testing purposes only._"

vaillant commented on content/service-terms/license-metrics-bundle/v1.md:
```diff
@@ -0,0 +1,110 @@
+* Dedicated Deployments have dedicated resources provided for a single Customer. Each Dedicated Deployment includes up to 3 nodes of Cumulocity IoT Database Enterprise Edition, where each node is limited to a physical or virtual machine that has up to 128 GB of RAM allocated to such node. Cumulocity IoT Database Enterprise Edition contains third party technology and the following additional licen ...
+* Public Cloud Deployments have shared resources provided for Customers. A Public Cloud Deployment will not have data shared between Customers. 
+
+Cumulocity Dedicated Deployments can be used for Development or Testing ("Non-Production") purposes but are restricted in the scope of their usage. Dedicated Deployments may be used by the Customer solely for non-production purposes only and may not be used for the processing of live data in a production environment or any purposes other than development or testing purposes only. A Non-Production ...
```
"Note that Cumulocity Public Cloud Deployments are always considered" 
That is actually not the case for eu-latest.

tydunne replied:
For billing purposes, EU-latest is considered a "production" environment. We don't formally offer "non-production" public cloud. If a customer wants a tenant on EU-Latest, that is fine, but we still maintain control over the release cycle.

vaillant commented on content/service-terms/license-metrics-bundle/v1.md:
```diff
@@ -0,0 +1,110 @@
+1. The total count of all data transactions to and in the Cumulocity platform, or
+2. The total count of messages processed through the MQTT Service.
+
+A data transaction is defined as any authenticated API request with the intention to create, update, or process data within the Cumulocity IoT domain model, including inventory objects, operations, measurements, events, and alarms. Transactions are counted regardless of processing mode, the details of which can be found in the [Tecnical Concept section of the Cumulocity documentation](https://cum ...
```
"Tecnical" => Technical

BeateRixen replied:
Fixed

vaillant commented on content/service-terms/license-metrics-bundle/v1.md:
```diff
@@ -0,0 +1,110 @@
+
+All counted transactions and messages are subject to the quotes listed in the [Service Quotas](https://cumulocity.com/docs/service-terms/quotas/) section of the Cumulocity documentation. Usage that exceeds the listed limits may result in additional charges to ensure platform stability and prevent abuse.
+
+For each billing period, the sum of daily MQTT message counts and the total count of data transactions are separately calculated. The greater of the two values is used as the final invoiced metric, rounded up to the nearest 100,000 units. In the case that one of the values is unable to be computed, the other value will be used for billing purposes. 
```
Minor: Somehow the different wording for MQTT and data transaction is irritating, Not sure how to make it simpler.

vaillant commented on content/service-terms/license-metrics-bundle/v1.md:
```diff
@@ -0,0 +1,110 @@
+
+## GiB for Operational Data Store
+
+The total volume of data stored in the Operational Data Store of the Cumulocity platform within a given billing period (calendar month). ODS is counted and exposed via the [Usage Statistics API](https://cumulocity.com/api/core/#tag) provided by the platform. The maximum daily value taken during the billing period will be used for invoicing purposes and is rounded up to the nearest gibibyte (GiB). ...
```
Minor: ODS is not really introduced => Add (ODS) after first introduction, or IMHO even better, don't use it at all.

tydunne replied:
I will remove the acronym!

vaillant commented on content/service-terms/license-metrics-bundle/v1.md:
```diff
@@ -0,0 +1,110 @@
+
+## Cumulocity Compute Unit (CCU)
+
+A Cumulocity Compute Unit (CCU) represents a standardized measure of computational resources consumed by microservices developed and deployed by the Customer, where 1 CCU is equivalent to 1 CPU core and 4 gibibytes (GiB) of memory. CCUs are calculated monthly by computing the daily average CPU consumption and daily average memory consumption (in bundles of 4 GiB) and taking the larger of the two  ...
```
I guess this is old text, but "consumption" is not that great. 
(Below it is detailed based on "resource limits").
Replace with "resource"?

tydunne replied:
Good point, I will replace "consumption" with "resource" - this is more accurate anyway.

vaillant commented on content/service-terms/license-metrics-bundle/v1.md:
```diff
@@ -0,0 +1,110 @@
+
+## DataHub - Memory Unit
+
+The total amount of memory available for use by DataHub nodes. A node is a physical or virtual machine or a container within the DataHub cluster that operates as either a co-ordinator node and/or an executor node. Both coordinator and executor nodes are required to run the DataHub Add-On. A DataHub node must be deployed with no less than 16 GB of memory. The memory allocation will be split betwee ...
```
"A DataHub node must be deployed with no less than 16 GB of memory" 
Does this match with the followint statement which requires min 32GB?

tydunne replied:
Only the first set needs to be deployed with 32 GB of memory. After that, 16 GB nodes are fine.

vaillant commented on content/service-terms/license-metrics-bundle/v1.md:
```diff
@@ -0,0 +1,110 @@
+
+## Virtual Private Connection
+
+Use by the Customer of the Cloud Services whose License Metric is indicated as “Virtual Private Connection” above allows the creation of a total number of said connections via a redundant pair of site-to-site ("S2S") tunnels between the Customer and the Supplier which does not exceed the licensed number indicated above. The reference to “Virtual Private Connection” refers to the use of a single r ...
```
I do not really understand the paragraph:
"Above" refers to what?
Very long sentence with many new terms.
Even with my (OK, limited) VPN knowledge I do not understand it.

tydunne replied:
This definition was taken directly from the existing contracts - I missed the "above" statement, which refers to the bill of materials. I'll tweak it and send to Oliver/Thomas for review.

BeateRixen committed:
Review changes for spelling, shortcodes, headings

BeateRixen committed:
Update v1.md

BeateRixen committed:
Update v1.md

BeateRixen committed:
Update v1.md

BeateRixen commented:
@tydunne I fixed the bundle issue, the layout looks good now. Moreover I did a language review directly in your file and fixed spelling issues/inconsistencies, added short codes, fixed heading levels, and more. Please check again.

tydunne committed:
Update v1.md

updated non-production language

tydunne committed:
Update v1.md

updated to use "Dedicated Cloud" instead of "Dedicated" to match with "Public Cloud"

tydunne committed:
Update v1.md

removed ODS acronym

tydunne committed:
Update v1.md

changed "consumption" to "resources" in the custom CCU definition.

tydunne commented:
> @tydunne I fixed the bundle issue, the layout looks good now. Moreover I did a language review directly in your file and fixed spelling issues/inconsistencies, added short codes, fixed heading levels, and more. Please check again.
> 
> Also please check Stefans comments as the refer to the content.

Hi @BeateRixen, thanks very much for fixing it. One comment - 
![image](https://github.com/user-attachments/assets/ff474736-7fa1-442d-ba89-81e2be8d8ece)
Is there a way we can make the page say "Version 1" instead of "License Metrics-Version 1"?

BeateRixen commented on content/service-terms/license-metrics-bundle/index.html:
```diff
@@ -0,0 +1,4 @@
+---
+title: Service-level agreements
```
```suggestion
title: License metrics
```

BeateRixen commented on content/service-terms/license-metrics-bundle/v1.md:
```diff
@@ -0,0 +1,108 @@
+---
+title: License metrics - Version 1
```
```suggestion
title: Version 1
```

BeateRixen commented on content/service-terms/license-metrics-bundle/v1.md:
```diff
@@ -0,0 +1,108 @@
+### Deployment
+
+A unique instance of the hosted software ("Software") provided by the supplier ("Supplier") and made accessible to the Customer. Deployments can be "Dedicated Cloud" instances or "Public Cloud" instances.
+* Dedicated Cloud deployments have dedicated resources provided for a single Customer. Each Dedicated Cloud deployment includes up to 3 nodes of {{< company-c8y >}} Database Enterprise Edition, where each node is limited to a physical or virtual machine that has up to 128 GB of RAM allocated to such node. {{< company-c8y >}} Database Enterprise Edition contains third-party technology and the foll ...
```
```suggestion
* Dedicated Cloud deployments have dedicated resources provided for a single Customer. Each Dedicated Cloud deployment includes up to 3 nodes of {{< company-c8y >}} Database Enterprise Edition, where each node is limited to a physical or virtual machine that has up to 128 GB of RAM allocated to such node. {{< company-c8y >}} Database Enterprise Edition contains third-party technology and the following additional license restrictions apply: The Supplier will inform the Customer in the event of termination of the agreement between the third-party technology provider and the Supplier. Customer's license to the third-party technology within {{< company-c8y >}} Database Enterprise Edition will automatically revert to the server-side public license unless the Customer and third-party technology provider agree otherwise, for example, by obtaining a license. The Software will still be suitable for the customer-specific use case in either case. If the Customer does not comply with the preceding, the Customer must remove the {{< company-c8y >}} Database Enterprise Edition from the Software. If the Customer is an agency, department, or other entity of the United States government, the use, duplication, reproduction, release, modification, disclosure, or transfer of the third-party technology, or any related documentation of any kind, including technical data and manuals, is restricted by a license agreement or by the terms of this contract in accordance with United States Federal Acquisition Regulation 12.212 for civilian purposes and United States Defense Federal Acquisition Regulation Supplement 227.7202 for military purposes. The third-party technology was developed fully at private expense. All other use is prohibited.
```

BeateRixen commented on content/service-terms/license-metrics-bundle/v1.md:
```diff
@@ -0,0 +1,108 @@
+
+A unique instance of the hosted software ("Software") provided by the supplier ("Supplier") and made accessible to the Customer. Deployments can be "Dedicated Cloud" instances or "Public Cloud" instances.
+* Dedicated Cloud deployments have dedicated resources provided for a single Customer. Each Dedicated Cloud deployment includes up to 3 nodes of {{< company-c8y >}} Database Enterprise Edition, where each node is limited to a physical or virtual machine that has up to 128 GB of RAM allocated to such node. {{< company-c8y >}} Database Enterprise Edition contains third-party technology and the foll ...
+* Public Cloud deployments have shared resources provided for Customers. A Public Cloud Deployment will not have data shared between Customers.
```
```suggestion
* Public Cloud deployments have shared resources provided for Customers. A Public Cloud deployment will not have data shared between Customers.
```

BeateRixen commented on content/service-terms/license-metrics-bundle/v1.md:
```diff
@@ -0,0 +1,108 @@
+* Dedicated Cloud deployments have dedicated resources provided for a single Customer. Each Dedicated Cloud deployment includes up to 3 nodes of {{< company-c8y >}} Database Enterprise Edition, where each node is limited to a physical or virtual machine that has up to 128 GB of RAM allocated to such node. {{< company-c8y >}} Database Enterprise Edition contains third-party technology and the foll ...
+* Public Cloud deployments have shared resources provided for Customers. A Public Cloud Deployment will not have data shared between Customers.
+
+{{< product-c8y-iot >}} Dedicated Cloud deployments can be used for development or testing ("Non-Production") purposes but are restricted in the scope of their usage. In this case, the Dedicated Cloud deployment may not be used for the processing of live production data or any purposes other than development or testing purposes only. A Non-Production Dedicated Cloud deployment may be subject to s ...
```
```suggestion
{{< product-c8y-iot >}} Dedicated Cloud deployments can be used for development or testing ("Non-Production") purposes but are restricted in the scope of their usage. In this case, the Dedicated Cloud deployment may not be used for the processing of live production data or any purposes other than development or testing purposes only. A Non-Production Dedicated Cloud deployment may be subject to separate fees as written in the contract between {{< company-c8y >}} and the Customer. Note that {{< product-c8y-iot >}} Public Cloud deployments are always considered production deployments, regardless of any usage intentions by the Customer.
```

BeateRixen commented on content/service-terms/license-metrics-bundle/v1.md:
```diff
@@ -0,0 +1,108 @@
+1. The total count of all data transactions to and in the {{< product-c8y-iot >}} platform, or
+2. The total count of messages processed through the MQTT Service.
+
+A data transaction is defined as any authenticated API request with the intention to create, update, or process data within the {{< product-c8y-iot >}} domain model, including inventory objects, operations, measurements, events, and alarms. Transactions are counted regardless of processing mode, the details of which can be found in the [Technical Concept section of the {{< product-c8y-iot >}} doc ...
```
```suggestion
A data transaction is defined as any authenticated API request with the intention to create, update, or process data within the {{< product-c8y-iot >}} domain model, including inventory objects, operations, measurements, events, and alarms. Transactions are counted regardless of processing mode, the details of which can be found in the [Technical concepts section of the {{< product-c8y-iot >}} documentation](https://{{< domain-c8y >}}/docs/concepts/realtime/#implementation). Transactions originate from both internal and external sources, including other {{< product-c8y-iot >}} applications, and are recorded via the platform’s [Usage Statistics API](https://{{< domain-c8y >}}/api/core/#tag/Usage-statistics).
```

BeateRixen commented on content/service-terms/license-metrics-bundle/v1.md:
```diff
@@ -0,0 +1,108 @@
+
+The total count of MQTT messages is metered daily via the MQTT Service, with each day's count contributing to the cumulative total for the billing period.
+
+All counted transactions and messages are subject to the quotes listed in the [Service quotas](https://{{< domain-c8y >}}/docs/service-terms/quotas/) section of the {{< product-c8y-iot >}} documentation. Usage that exceeds the listed limits may result in additional charges to ensure platform stability and prevent abuse.
```
```suggestion
All counted transactions and messages are subject to the quotes listed in [Service quotas](https://{{< domain-c8y >}}/docs/service-terms/quotas/). Usage that exceeds the listed limits may result in additional charges to ensure platform stability and prevent abuse.
```

BeateRixen committed:
Apply suggestions from code review

BeateRixen commented:
> > @tydunne I fixed the bundle issue, the layout looks good now. Moreover I did a language review directly in your file and fixed spelling issues/inconsistencies, added short codes, fixed heading levels, and more. Please check again.
> > Also please check Stefans comments as the refer to the content.
> 
> Hi @BeateRixen, thanks very much for fixing it. One comment - ![image](https://private-user-images.githubusercontent.com/175710376/431978195-ff474736-7fa1-442d-ba89-81e2be8d8ece.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDQyNzM5NzYsIm5iZiI6MTc0NDI3MzY3NiwicGF0aCI6Ii8xNzU3MTAzNzYvNDMxOTc4MTk1LWZmNDc0NzM2LTdmYTEtNDQyZC1iYTg5LTgxZTJiZThkOGVjZS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwNDEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDQxMFQwODI3NTZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xNDJjN2EwNTYzODJmYmIzOWU4MzI4YjE5MGFlYmNhZDNiY2ViM2IwZjkxYTgwMzgxMmYzNjJiNjc5ZmVjNDRmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.rXESw9z4LHuSERddzct9cncg_vDi2nzak22Nm0PQNkA) Is there a way we can make the page say "Version 1" instead of "License metrics-Version 1"?

Changed to Version 1.

eickler commented:
While it's good to have the concrete definitions, the documents is in some places a bit odd if you read it from the perspective of a cloud service customer. Haven't seen anything like that anywhere for SaaS.

Lifecycle and versioning: Lifecycle and versioning should be defined for the entire terms of service, not only for the billing metrics. We earlier agreed to have only a latest version published on the web site, because we anyway can't calculate different metrics for different customers or provide different services for different customers who purchased the cloud product. So I would remove the "Version 1", make it a top-level document in the section and also remove the comment about the timeline. 

Cloud service vs internal hardware resources, licenses etc.: Most of the "Deployment" section makes little sense for a cloud service (and I take it that this page is limited to the cloud service). System dimensioning, internal architecture, internal third party licensing are "behind" the cloud service and usually not exposed to the customer. The billing metrics plus the limits documentation should cover whatever constraints we have in terms of dimensioning and licensing. The section about the third party technology provider is likely not comprehensible to customers and uses terms that are not defined beforehand ("server-side public license"). I would remove all sentences that are not related to legal disclaimers or direct end user usage constraints.

Readability: Customers probably appreciate a gentle introduction section on what the page is about. Even I was initially confused why there was the "replica set" section on the page until I noticed that it's actually an option that we sell. (And again, it's an odd option for a cloud service.) The intro section could also introduce the concept of gibibyte for the entire remainder, so that you don't need to repeat it all the time. (E.g, "except if otherwise stated, all GB metrics are in gibibyte" or something.) 

Focus: Maybe focus this page really only on billing metrics and put other constraints outside of the metrics elsewhere. The "sharing" disclaimer is IMHO sufficiently covered by Section "shared environment considerations" in the SLA above. 

There is a conlifct:
A “gibibyte” is defined as 1,073,741,824 bytes.
A “gibibyte” is defined as 1,000,000,000 bytes.
If you use both Gib and GB, it can be defined at the top and consistently used throughout the page.

tydunne commented:
> While it's good to have the concrete definitions, the documents is in some places a bit odd if you read it from the perspective of a cloud service customer. Haven't seen anything like that anywhere for SaaS.
> 
> Lifecycle and versioning: Lifecycle and versioning should be defined for the entire terms of service, not only for the billing metrics. We earlier agreed to have only a latest version published on the web site, because we anyway can't calculate different metrics for different customers or provide different services for different customers who purchased the cloud product. So I would remove the "Version 1", make it a top-level document in the section and also remove the comment about the timeline.
> 
> Cloud service vs internal hardware resources, licenses etc.: Most of the "Deployment" section makes little sense for a cloud service (and I take it that this page is limited to the cloud service). System dimensioning, internal architecture, internal third party licensing are "behind" the cloud service and usually not exposed to the customer. The billing metrics plus the limits documentation should cover whatever constraints we have in terms of dimensioning and licensing. The section about the third party technology provider is likely not comprehensible to customers and uses terms that are not defined beforehand ("server-side public license"). I would remove all sentences that are not related to legal disclaimers or direct end user usage constraints.
> 
> Readability: Customers probably appreciate a gentle introduction section on what the page is about. Even I was initially confused why there was the "replica set" section on the page until I noticed that it's actually an option that we sell. (And again, it's an odd option for a cloud service.) The intro section could also introduce the concept of gibibyte for the entire remainder, so that you don't need to repeat it all the time. (E.g, "except if otherwise stated, all GB metrics are in gibibyte" or something.)
> 
> Focus: Maybe focus this page really only on billing metrics and put other constraints outside of the metrics elsewhere. The "sharing" disclaimer is IMHO sufficiently covered by Section "shared environment considerations" in the SLA above.
> 
> There is a conlifct: A “gibibyte” is defined as 1,073,741,824 bytes. A “gibibyte” is defined as 1,000,000,000 bytes. If you use both Gib and GB, it can be defined at the top and consistently used throughout the page.

Hello André, 

Thank you for your detailed review. I will respond to each note.

Lifecycle and versioning: 
- The License Metrics were recommended to be removed from our contracts and placed online by our General Counsel, Alexander. Previously, license metrics were included in every single contract as there was a defined bill of materials that needed the license metrics. Now that we are moving away from the bill of materials, we have the opportunity to simplify our contracts. Alexander also mentioned that the license metrics are too technical to be in our contracts.
- We cannot have one single version for license metrics, as license metrics need to be static for the duration of a contract. We cannot legally update the metrics while a contract references them. As such, a versioning approach is necessary. In some cases where the customer requires it, the latest version of the page may be exported and appended to contracts directly. 
- Again, this was the recommended approach by our lawyer. 

Cloud service vs internal hardware resources, licenses etc.:
- All of the MongoDB language is not my own, but rather came directly from the Software AG TCA team, who stated we must include it in our contracts in order to be compliant with our contract with MongoDB. Since we are removing license metrics from contracts, we need to include it here. 
- I agree it is an eyesore, but I don't have a better solution for where it needs to go (since it is required language by MongoDB). 

Readability:
- That is a good idea - I will add in an introduction explaining what these metrics are, how the versioning works, etc. 
- I'll put the definition of a Gibibyte in that introduction (I don't anticipate that it will change between versions). 

Focus:
- What is the "sharing disclaimer" you refer to? Is it just the definition of the Public Cloud? I am happy to change that, what do you recommend I write instead?

Conflict:
- I've now confirmed with the Ops and DataHub team that they use the "Gibibyte" definition so I can safely remove the discrepancy.

tydunne committed:
Create intro.md

add introduction

tydunne committed:
Update v1.md

updated introduction, corrected "gibibyte" definitions

tydunne committed:
Update intro.md

added note about validity only for CTC contracts

tydunne committed:
Update v1.md

removed MongoDB statement

eickler reviewed (approved):
I like the intro, it's also good to cover the applicability there.

One issue still: There is one conflict in the definitions, can we still fix that? Does it require discussion?

On the license metrics page:

> Note that Cumulocity Public Cloud deployments are always considered production deployments, regardless of any usage intentions by the Customer.

On the release policy page:

> Non-production instances include customer development and test instances, and the public instance at eu-latest.cumulocity.com.
> Production instances cover both customer production environments and other public cloud production instances.

(That's also how we treat eu-latest from ops perspective.)

tydunne committed:
Update v1.md

changed public cloud non prod text

tydunne commented:
As discussed with Andre over chat, I've changed that sentence to "Note that Cumulocity Public Cloud deployments are always rated at the same rate regardless of instance type." and we should be good to go!

@BeateRixen - please merge this when you are able. Thank you all!

BeateRixen commented on content/service-terms/license-metrics-bundle/intro.md:
```diff
@@ -0,0 +1,16 @@
+
+---
+
+In this section, the billable license metrics for each component of Cumulocity are defined. These metrics are used for billing purposes and represent how Cumulocity computes each unit during the billing period. License metrics are used to quantify resource consumption, system usage, and service utilization for accurate and transparent billing calculations. Customers should refer to these definiti ...
```
```suggestion
In this section, the billable license metrics for each component of {{< product-c8y-iot >}} are defined. These metrics are used for billing purposes and represent how {{< product-c8y-iot >}} computes each unit during the billing period. License metrics are used to quantify resource consumption, system usage, and service utilization for accurate and transparent billing calculations. Customers should refer to these definitions to understand how their usage of {{< company-c8y >}} services is measured and billed.
```

BeateRixen commented on content/service-terms/license-metrics-bundle/intro.md:
```diff
@@ -0,0 +1,16 @@
+
+Each version of the license metrics will be written with an effective date range. License metrics will not be updated during the duration of an active contract, and the version that is used upon signature will persist even if there are additional versions released afterwards. The start date for a contract will be used as the effective date for license metrics, and the version which is active at t ...
+
+The license metrics contained within this section are valid for "Commit-to-Consume" contracts and are not applicable to other contract types. To identify if your contract is a "Commit-to-Consume" contract, please follow up with your Cumulocity contract or view the contract directly. 
```
```suggestion
The license metrics contained within this section are valid for "Commit-to-Consume" contracts and are not applicable to other contract types. To identify if your contract is a "Commit-to-Consume" contract, please follow up with your {{< company-c8y >}} contract or view the contract directly. 
```

BeateRixen commented on content/service-terms/license-metrics-bundle/intro.md:
```diff
@@ -0,0 +1,16 @@
+
+The license metrics contained within this section are valid for "Commit-to-Consume" contracts and are not applicable to other contract types. To identify if your contract is a "Commit-to-Consume" contract, please follow up with your Cumulocity contract or view the contract directly. 
+
+For the purposes of all license metrics, one “gibibyte” is defined as 1,073,741,824 bytes as is abbreviated 'GiB'. A gibibyte may be colloqually referred to as a gigabyte and these names should be treated as the same value.
```
```suggestion
{{< c8y-admon-info >}}
For the purposes of all license metrics, one “gibibyte” is defined as 1,073,741,824 bytes as is abbreviated 'GiB'. A gibibyte may be colloqually referred to as a gigabyte and these names should be treated as the same value.
{{< /c8y-admon-info >}}
```

BeateRixen committed:
Apply suggestions from code review
